.github/workflows/python.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         working-directory: ./bindings/python
         run: |
           source .env/bin/activate
-          pip install black==20.8b1 click==8.0.4
+          pip install black==22.3 click==8.0.4
```

Comment:
would it make sense to make them [setup tools extras](https://github.com/huggingface/doc-builder/blob/main/setup.py#L12) & call as `pip install tokenizers[dev]` ?

Reply:
Hmm it does, just moves the location of this information. Will have to check about `click` too.